src/routes/(console)/organization-[organization]/billing/planSummary.svelte

```
@@ -144,6 +149,21 @@
         );
     }
 
+    import { page } from '$app/state';
+
+    let projectsLimit: number = 5;
+    let projectsOffset: number = 0;
+    $: projectsLimit = limit ?? (Number(page.url.searchParams.get('limit')) || 5);
+    $: projectsOffset =
+        offset ?? ((Number(page.url.searchParams.get('page')) || 1) - 1) * projectsLimit;
+    $: totalProjects =
+        (currentAggregation && (currentAggregation as any).breakdownTotal) ||
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)